### PR TITLE
SIG ML Dev Exp charter addition

### PR DESCRIPTION
**Pull request by jkoehler-redhat**

### CODE REVIEW

**Comment by jkoehler-redhat** [comment]
I checked the code via VS code and the files looked good
Screen Shot 2022-05-12 at 3 48 13 PM
Screen Shot 2022-05-12 at 3 49 00 PM

File: sig-ml-developer-experience/README.md
Comment (anishasthana) [inline sig-ml-developer-experience/README.md:13]: Possibly dumb question - do we want to include the GitHub handles for all the members here?

Comment (anishasthana) [reply sig-ml-developer-experience/README.md:13]: Both under chairs and members

**Comment by shgriffi** [comment]
This PR is missing the owners file (see https://github.com/shgriffi/opendatahub-community/blob/master/sig-data-scientist-experience/OWNERS).  @anishasthana that will at least cover what you mentioned about the github handles.  But I agree, might be easy enough to have them in the SIG charter too.

@jkoehler-redhat can we get the OWNERS file added?

**Comment by jkoehler-redhat** [comment]
@shgriffi @anishasthana i did not add that file just yet as i was not sure who would be considered owners/reviewers.  I am assuming we would need to have all the repos ID'd?  i was gonna let the sig leads help with that determination.  let me know what you think i should do here.

**Review by anishasthana — COMMENTED** [review]
Overall this looks good to me!
I'll defer to Sherard/Landon on final review.

**Comment by shgriffi** [comment]
@jkoehler-redhat can you add periods at the end of all of the sentences in the README?  After that, I'm good for this to be merged.

**Review by LaVLaS — APPROVED** [review]
/lgtm

**Pull request merged**
